pkg/controller/etcdproxy/helpers.go

```diff
@@ -19,7 +19,8 @@ func newReplicaSet(etcdstorage *etcdstoragev1alpha1.EtcdStorage,
 	etcdControllerNamespace, etcdProxyNamespace, etcdProxyImage,
 	etcdCoreCAConfigMapName, etcdCoreCertSecretName string, etcdCoreURLs []string) *appsv1.ReplicaSet {
 	labels := map[string]string{
-		"controller": "epc",
+		"controller": "etcdproxy",
```

Comment:
aren't there standard labels used elsewhere?

Reply:
You mean the `epc` label? No, it's used nowhere, but I was putting it to be used by service.
As it's not used by service at all, should we remove it maybe?

Reply:
> aren't there standard labels used elsewhere?

@sttts I don't know what you're referring to either.

Reply:
Some standardized label or annotation to express which controller this belongs to.